README.md

```diff
@@ -79,142 +54,165 @@ $ open QuickStart.xcworkspace
 ```
 
 -	In the QuickStart project, open the plist file `settings.plist`.  Replace the values of the elements in the section to reflect the values you input into the Azure Portal.  Your code will reference these values whenever it uses ADAL.
-    -	The `tenant` is the domain of your Azure AD tenant, e.g. contoso.onmicrosoft.com
     -	The `clientId` is the clientId of your application you copied from the portal.
     -	The `redirectUri` is the redirect url you registered in the portal.
+    -   The `graphAPI` is the Graph API we will be calling in the QuickStart. Here it is http://graph.microsoft.com
+    -   The `api-version` is the API version of the Microsoft Graph API you wish to call.
```

Comment:
Consider linking to any description of the various versions and when to use each.